### PR TITLE
Add sdk versions to integration.md

### PR DESCRIPTION
**Pull request by alpe**
Doc update

### CODE REVIEW

File: INTEGRATION.md [hunk 1]
```diff
@@ -12,19 +14,23 @@ The pre-requisites of integrating `x/wasm` into your custom app is to be using
...
-| v0.12 | v0.40.0-rc3 |
-| v0.11 | v0.39.1 |
-| v0.10 | v0.39.1 |
-| v0.9  | v0.38.3 |
-| v0.8  | v0.38.3 |
-| v0.7  | v0.38.3 |
+| wasmd  | Cosmos SDK  |
+|:------:|:-----------:|
+| v0.22  |   v0.45.0   |
+| v0.20  |   v0.42.x   |
```
Comment (ethanfrey) [inline INTEGRATION.md:20]: You skipped v0.21

**Review by ethanfrey — COMMENTED** [review]
I think we have another doc with cosmwasm-vm compatibility.
It would be good to update that here as well.

**Comment by alpe** [comment]
👍 Updated the compatibility tables

**Comment by ethanfrey** [comment]
> 👍 Updated the compatibility tables

I think you didn't push

**Review by ethanfrey — APPROVED** [review]
Now I see it,.
Good stuff

**Pull request merged**